presto-docs/src/main/sphinx/security/password-validation.rst

```diff
@@ -0,0 +1,42 @@
+===========================
+Password Validation Changes
```

Comment:
```suggestion
Password Validation
```
Also, please change the length of the formatting header lines above and below to match the length of this line after editing.